mkdocs.yml

```diff
@@ -8,6 +8,11 @@ edit_uri: "edit/main/docs/"
 copyright: Copyright &copy; 2024 Alraune Zech, Sona Aseyednezhad, Robin Richardson, Jaro Camphuijsen
 
 nav:
+  - Getting started:
+    - Getting started: index.md
+    - Development: getting-started/development.md
```

Comment:
This file does not exist anymore. Also the menu item was moved to line 20 of this file, so this line can be removed. The same counts for index.md, as it is already put in the main menu item called "Home" in line 16. This in fact results in this whole main menu item "Getting started:" being redundant, so I suggest we remove these three lines. 
```suggestion
```